Add the mev-boost call for testing (#23)
flashbots/flashbots-writings-website: writings/2022-06-20-mev-boost-call-for-testing.mdx

> ---

Reviewer: post needs an OG image, like https://github.com/flashbots/flashbots-writings-website/blob/main/writings/2022-06-07-why-run-mevboost.mdx

Author: What's OG?

Reviewer: opengraph, it's the image that's shown when sharing a post. https://ogp.me/ and see also https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#opengraph

Author: I'm very bad at making things look nice. Is there a place to get good opengraph images?
Until we have a designer assigned to our communications, I would prefer to just go with whatever is the default on the writings website.

Reviewer: You can just use any free image, but without it the preview will not look good, there's no default

Reviewer: - [https://pixabay.com/](https://pixabay.com)
- [https://unsplash.com/](https://unsplash.com/)

slug: mev-boost-call-for-testing

> title: Boost your MEV in Ethereum proof-of-stake

Author: yes, I'm trying catchy on this one. The next ones will be more plain.

authors: [elopio]
tags: [mev-boost]

> hide_table_of_contents: false

Reviewer: ```suggestion
hide_table_of_contents: true
```

---

# Boost your MEV in Ethereum proof-of-stake

> Calling all validators to test `mev-boost` in the Kiln test network.

Reviewer: ```suggestion
Calling all validators to test [mev-boost](https://github.com/flashbots/mev-boost) in the Kiln and Ropsten test networks.
```

Author: I've added the link. About ropsten, let me se if I will have time to test both guides today.
Otherwise we can make a new call for testing with ropsten next week.

<!--truncate-->

From our [early explorations in Ethereum proof-of-stake](https://writings.flashbots.net/research/mev-eth2) we found that “MEV can increase validator rewards by 75.3%, or give an APR of 12.86% rather than a non-MEV APR of 7.35% from staking eth.” Any competitive validator will need access to MEV.

Introducing `mev-boost`, which gives all validators the ability to participate in MEV extraction. Using `mev-boost` validators can sell their blockspace to specialized block builders. We developed it in collaboration with the Ethereum developers and researchers. For more information, see [Why run mev-boost?](https://writings.flashbots.net/writings/why-run-mevboost/)

> Today, mev-boost is ready for testing, and we need you! If you’re a solo validator or node operator, *please help us test `mev-boost` on the Kiln test network*. Let us work together so the infrastructure is ready for the merge, and validators can profit from MEV from their first block.

Reviewer: ```suggestion
Today, mev-boost is ready for testing, and we need you! If you’re a solo validator or node operator, **please help us test `mev-boost` on the Kiln and Ropsten test networks**. Let us work together so the infrastructure is ready for the merge, and validators can profit from MEV from their first block.
```

Reviewer 2: *

> Follow this guide to run teku+nethermind+mev-boost in Kiln:

Reviewer: ```suggestion
Follow this guide to run Teku + Nethermind + mev-boost:
```

> https://github.com/flashbots/mev-boost/wiki/Testing#tekunethermindmev-boost

Author: I will make a shorter call for testing with prysm next week. And one for lighthouse once that's good too. More posts, more noise.

> https://github.com/flashbots/mev-boost/wiki/Testing#tekunethermindmev-boost

Author: The guide will not assume any prior knowledge. Well, ubuntu command line proficiency only. That way we can get new contributors in the ecosystem on-boarding through flashbots.
We can make a shorter guide, that can live in the readme.

Please test it and provide feedback as [issues](https://github.com/flashbots/mev-boost/issues). We will be sharing more guides as the other [Consensus Clients finish their implementations of the API](https://github.com/flashbots/mev-boost#consensus-clients-implementation-status). The roadmap, expected deliveries and estimated deadlines are described in [The Plan](https://github.com/flashbots/mev-boost/wiki/The-Plan-(tm)). Join us in the [`mev-boost` repository](https://github.com/flashbots/mev-boost/) while we explore the remaining [open research questions](https://github.com/flashbots/mev-boost/wiki/Research#open-questions) with all the relevant organizations in the ecosystem. If you have further questions, take a look at the [Frenquently Asked Questions](https://github.com/flashbots/mev-boost/wiki/Frequently-Asked-Questions).

---

> We thank all the contributors to this project from the consensus clients, Ethereum Foundation, the [Eth2 working group](https://writings.flashbots.net/writings/announcing-eth2-working-group/), and other organizations from the ecosystem. In particular, we appreciate [lightclient](https://github.com/lightclient/) for his leadership on the specification definition.

Reviewer: ```suggestion
We thank all the contributors to this project from the consensus clients, Ethereum Foundation, the [Eth2 Working Group](https://writings.flashbots.net/writings/announcing-eth2-working-group/), and other organizations from the ecosystem. In particular, we appreciate [lightclient](https://github.com/lightclient/) for his leadership on the specification definition.
```

---
slug: mev-boost-call-for-testing
title: Boost your MEV in Ethereum proof-of-stake
authors: [elopio, metachris]
tags: [mev-boost]
image: img/dall-e.png
hide_table_of_contents: false
---

# Boost your MEV in Ethereum proof-of-stake

Calling all validators to test [`mev-boost`](https://github.com/flashbots/mev-boost).

<!--truncate-->

From our [early explorations in Ethereum proof-of-stake](https://writings.flashbots.net/research/mev-eth2) we found that “MEV can increase validator rewards by 75.3%, or give an APR of 12.86% rather than a non-MEV APR of 7.35% from staking eth.” Any competitive validator will need access to MEV.

> Introducing `mev-boost`, which gives all validators the ability to participate in MEV extraction. Using `mev-boost` validators can sell their blockspace to specialized block builders. We developed it in collaboration with the Ethereum developers and researchers. For more information, see [Why run mev-boost?](https://writings.flashbots.net/writings/why-run-mevboost/)

Reviewer: ```suggestion
MEV-Boost gives all validators the ability to participate in MEV extraction by selling their blockspace to specialized block builders. We developed mev-boost it in collaboration with the Ethereum developers and researchers. For more information, see [Why run mev-boost?](https://writings.flashbots.net/writings/why-run-mevboost/)
```
